packages/account/src/Sections/Security/ConnectedApps/__tests__/connected-apps.spec.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { render, screen, waitFor, waitForElementToBeRemoved, fireEvent } from '@testing-library/react';
+import ConnectedApps from '../connected-apps';
+import { act } from 'react-dom/test-utils';
```

Comment:
It is recommended to use `act` from `@testing-library/react`:
https://testing-library.com/docs/react-testing-library/api/#act